en-US/firefox-ios.xliff

```diff
@@ -697,6 +686,11 @@
         <target>About</target>
         <note>About settings section title</note>
       </trans-unit>
+      <trans-unit id="ActivitySream.Pocket.Trending">
```

Comment:
@farhanpatel there is a spelling mistake here in the `id`. Do we want to fix that?

Reply:
Fixed in https://github.com/mozilla-mobile/firefox-ios/pull/3291